README.md

```diff
@@ -90,7 +90,155 @@ Commit this file to your Git repository as it ensures future builds to use the v
 When running your client code in development on port 8081, it will help to run the MOLGENIS locally on port 8080. 
 The vue-cli comes with a proxy table that will redirect any REST calls to localhost:8080.
 
-### Using yarn link for live editing
+### Testing packages
+When you want to test a package there are actually three ways to do this. 
+
+1. [Make a standalone setup and mock the API responses](#make-a-standalone-setup)
+2. [Proxy the MOLGENIS backend in front of the package](#proxying-molgenis-backend)
+3. [Link the package in the MOLGENIS backend for full integration](#using-yarn-link-for-live-editing) 
+
+> **note: the hierarchy of the ways to test the packages is important.**
+>
+>The first and second test manners should be sufficient to actually be confident that you application works. In the preview mode, 
+where you already integrated the artifact in MOLGENIS you also test the integration. 
+
+#### Make a standalone setup
+You always want to start with an offline setup for the package. You then need to define the state and mock 
+responses first and have to think about the API you want to use. In cli2 and cli3 the setup is different so 
+both are described.  
+
+##### cli2
```

Comment:
Is **cli2** still supported? In the section **Create new stable package** it is not mentioned.

Reply:
No that is correct. However we still have some cli2 apps/plugins. We do not want to create new ones with cli2. The vision is to move all apps/plugins/packages to cli3 over time. In the mean time we do want to know how we can use it for testing.

Reply:
Check review comments in files